Jenkinsfile

```diff
@@ -41,21 +41,21 @@ pipeline {
       }
     }
 
-    stage('deploy') {
-      when { branch 'master' }
-      steps {
-        sh 'git config --global user.email james.kassemi+vabot@adhocteam.us'
-        sh 'git config --global user.name va-bot'
-        sh 'git config --global credential.helper "/bin/bash ' + env.WORKSPACE + '/scripts/credential-helper.sh"'
-        withCredentials([[
-          $class: 'UsernamePasswordMultiBinding',
-          credentialsId: 'va-bot',
-          usernameVariable: 'GIT_USERNAME',
-          passwordVariable: 'GIT_PASSWORD'
-        ]]) {
-          sh "GITHUB_API_KEY=${env.GIT_PASSWORD} npm run deploy"
-        }
-      }
-    }
+   // stage('deploy') {
```

Comment:
Are we waiting to remove the other steps in Jenkins until after this is merged and confirmed working?

Reply:
I had the same thought too... but if we leave jenkins, doesnt it get deployed twice?

Reply:
Commenting the `deploy` step like you did should prevent it from being deployed twice. But I'm wondering if we still need the other jobs in the pipeline. Maybe we can comment those out too, since we won't need them. Either way, we'll probably need to revisit the Jenkinsfile at some point to remove it entirely from this repo

Reply:
oops I understood what you mean now "other steps" removal too :-)
Yes agree. I want to make sure GHA is all set before deleting everything, so first remove deploy and then the rest!